### ISSUE TITLE
Donation leaderboard

### ISSUE THREAD

**Issue by luisivan** [helpdao/help#8]
It would be cool if there was a donation leaderboard per help squad!

**Comment by yeqbfgxjiq** [comment]
This seems very similar to #17. What if we have a Contributors page (or section) that recognized all the awesome people who have donated time or money to help make HelpDAOs possible?

**Comment by luisivan** [comment]
Yes! But this one is specific to donors within independent help squads
